addons/binding/org.openhab.binding.gpstracker/ESH-INF/i18n/gpstracker_en.properties

```diff
@@ -0,0 +1,25 @@
+# binding
+binding.gpstracker.name = GPSTracker Binding
+binding.gpstracker.description = GPSTracker binding with support of OwnTracks and GPSLogger trackers over HTTP 
+
+# thing types
+thing-type.gpstracker.tracker.label = GPS Tracker
+thing-type.gpstracker.tracker.description = Tracker application running on mobile device
+
+# channel types
+channel-type.gpstracker.lastReport.label = Last report timestamp
+channel-type.gpstracker.lastReport.description = Timestamp of the last report received from tracker
+channel-type.gpstracker.location.label = Tracker location
+channel-type.gpstracker.location.description = Location sent by the tracker
+channel-type.gpstracker.batteryLevel.label = Battery level
+channel-type.gpstracker.batteryLevel.description = Battery level of the device running the tracker application
+channel-type.gpstracker.regionPresence.label = Presence switch
```

Comment:
Is this still used anywhere?

Reply:
The label is dynamic - this key is not used. But do we need the `channel-type.gpstracker.regionPresence.description`? I don't think I've seen it on screens...